src/main/java/com/openshift/restclient/model/build/BuildStrategyType.java

```diff
@@ -13,11 +13,14 @@
  * by OpenShift
  */
 
-// TODO: evalute switching to a class/constants since enums are not extendable
-public enum BuildStrategyType {
-	Docker,
+public interface BuildStrategyType {
+	
+	static final String Docker = "Docker";
```

Comment:
Something I didnt catch so far: 
Shouldnt name of public (final) constants be capitalized? http://stackoverflow.com/a/7259738/231357